mkosi/__init__.py

```diff
@@ -2241,6 +2242,100 @@ def install_openmandriva(args: CommandLineArguments, root: str, do_run_build_scr
     disable_pam_securetty(root)
 
 
+@complete_step("Installing Gentoo")
+def install_gentoo(args: CommandLineArguments, root: str, do_run_build_script: bool) -> None:
+    try:
+        import multiprocessing
+
+        from _emerge.actions import (
+            _emerge_config,
+            action_build,
+            action_sync,
+            load_emerge_config,
+        )
+
+        if "build-script" in args.debug:
+            from _emerge.actions import action_info
+    except ImportError:
+        die("You need portage module for Gentoo: https://gitweb.gentoo.org/proj/portage.git")
+
+
+    jobs = multiprocessing.cpu_count()
+
+    binpkgdir_suffix = args.hostname if args.hostname else "live"
+    pkgdir = os.path.join("/var/cache", "binpkgs-" + binpkgdir_suffix)
+    os.makedirs(pkgdir, exist_ok=True)
+
+    portdir = "/var/db/repos/gentoo"
+
+    os.environ["PORTAGE_CONFIGROOT"] = root
+    os.environ["SYSROOT"] = root
+    os.environ["ROOT"] = root
+    os.environ["EPREFIX"] = "/"
+    os.environ["PORTDIR"] = portdir
+    os.environ["PKGDIR"] = pkgdir
+    os.environ["KERNEL_DIR"] = os.path.join(root, "usr/src/linux")
+    # -userpriv is required since we're bind-mounted
+    # --buildpkg without FEATURES=buildpkg is no go!
+    os.environ["FEATURES"] = "-userpriv buildpkg parallel-install"
+
+    emerge_config = _emerge_config()
+    emerge_config = load_emerge_config(env=os.environ, args=[], opts={})
+    action_sync(emerge_config)
+
+    GENTOO_ARCHITECTURES = {
+        "x86_64": "amd64",
+        "aarch64": "arm64",
+    }
+
+    if args.architecture:
+        gentoo_arch = GENTOO_ARCHITECTURES.get(args.architecture, "amd64")
+
+    # this should let users hack this flag by passing *flavour* for their image
+    # as in --release=17.1/no-multilib
+    GENTOO_LATEST = "17.1"
```

Comment:
Can you set this in `load_args`? That's where all the other distro release defaults are located.